schema/workflow.json

```diff
@@ -1339,16 +1336,16 @@
           "description": "State end definition"
         },
         "inputCollection": {
-          "type": "string",
-          "description": "Workflow expression selecting an array element of the states data"
+          "$ref": "common.json#/definitions/arrayOrMarkup",
```

Comment:
why array or markup. the expression never returns an array. it just selects an array.

Reply:
Maybe you want to construct your array:
```
inputCollection: [
  "alwaysThis",
  "${ .someMoreElements }",
  "${ .different.path.to.some.moreElements }",
  "andAlsoThis"
]
```